backend/tags/views.py

```
@@ -20,6 +20,7 @@ class TagViewSet(viewsets.ModelViewSet):
     pagination_class = CustomPagination
     ordering_fields = ["created_at"]
     filter_backends = [DjangoFilterBackend, OrderingFilter]
+    filterset_fields = ["name"]
```

Ruff (0.15.13)

[warning] Mutable default value for class attribute

(RUF012)